runtime/autoload/health/nvim.vim

```diff
@@ -173,6 +173,11 @@ function! s:check_terminal() abort
     call health#report_info('key_dc (kdch1) terminfo entry: '
         \ .(empty(kbs_entry) ? '? (not found)' : kdch1_entry))
   endif
+  for env_var in ['XTERM_VERSION', 'VTE_VERSION', 'TERM_PROGRAM', 'COLORTERM', 'SSH_TTY']
+    if !empty(eval('$'.env_var))
```

Comment:
Why not `if !exists('$'.env_var)`?

Reply:
Ah, yes, that's more correct. For TUI purposes it doesn't matter, but I'll make that change.